crates/kornia-io/src/stream/capture.rs

```diff
@@ -160,8 +160,10 @@ impl StreamCapture {
             .ok_or_else(|| StreamCaptureError::GetBufferError)?
             .map_readable()?;
 
-        Image::<u8, 3>::new(ImageSize { width, height }, buffer.as_slice().to_vec())
-            .map_err(|_| StreamCaptureError::CreateImageFrameError)
+        let image = Image::from_raw_parts([width, height].into(), buffer.as_ptr(), buffer.len())
+            .map_err(|_| StreamCaptureError::CreateImageFrameError)?;
```

Comment:
@gbin here I'm getting. `buffer` doesn't won't the underlying memory

```bash
free(): invalid pointer
Aborted (core dumped)
```
to reproduce: `cargo run --bin webcam --release`